### DISCUSSION TITLE
Allow tab unloading

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#751] [Ideas]
### What feature would you like to see?

Similar to #709, I'd love to be able to see a feature for unloading a tab without closing it permanently. While Firefox has a built in mechanism for automatically unloading tabs, I sometimes prefer to unload them manually.

### Other details

Ideally, I think an addition to the tab context menu to unload the tab would work well, but perhaps there could be other settings for tab unloading.
 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/710_

**Comment by kwilcz** [comment]
I believe it's done in the latest update? @mauro-balades

**Reply by mr-cheffy** [reply]
Very true, thanks!

**Comment by adam-the** [comment]
Is this actually necessary? Doesn't [this extension](https://addons.mozilla.org/en-GB/firefox/addon/auto-tab-discard) allow management through Firefox's built-in mechanisms?

**Reply by kwilcz** [reply]
Extensions should not be an answer as their support might get dropped at any time.

**Reply by darkvergus** [reply]
this already exists since the last 2 versions. https://zen-browser.app/release-notes#1.0.1-a.6

![image](https://github.com/user-attachments/assets/2adbca22-f042-410b-996a-9c57bcbc53b3)

**Reply by adam-the** [reply]
> Extensions should not be an answer as their support might get dropped at any time.

They're typically replaced pretty quick too - I don't think loading tech debt onto a project with such a small team behind it is sensible.
The extension is arguably better implemented, perhaps even better than this could ever be implemented by Zen.

**Comment by Tealk** [comment]
Is there a way to protect tabs from unloading?

**Reply by darkvergus** [reply]
yes
![image](https://github.com/user-attachments/assets/7ec6b162-9ee6-43e1-a371-c437b4c8bf69)

**Reply by Tealk** [reply]
What is the difference between prevent and ignore?

**Reply by darkvergus** [reply]
frankly i have no idea

**Reply by Tealk** [reply]
I also can't see which option is currently active :/

**Reply by Tealk** [reply]
And no matter what I choose, it probably has no effect.

**Reply by kryst4line** [reply]
If you access about:unloads you can see the priority and usage of each tab... And yeah, neither option seem to disable tab unloading

**Comment by poiNt3D** [comment]
I would also like to know what is the difference between 2 options.
